### DISCUSSION TITLE
Let's Finally Settle the Kadi Game Rules!

### DISCUSSION THREAD

**Discussion by Jcardif** [Jcardif/Kadi#1] [Announcements]
Are we missing something on this set of rules? Put it here.

**Comment by BethanyJep** [comment]
Boxing day tunakaa chini kama country tuset rules standard za kadi

**Comment by BethanyJep** [comment]
Fines za ujinga can be administered when :
😸. Someone says "kadi" after their turn has already passed
😸. Picking back your card after you've already placed it down
😸. Cards are foolishly placed down without adhering to the rules of the game

**Comment by BethanyJep** [comment]
Fines za ujinga can be administered when :
😸. Someone says "cardi" , "kadi" after their turn has already passed
😸. Picking back your card after you've already placed it down
😸. Cards are foolishly placed down..without adhering to the rules of the game

**Comment by BethanyJep** [comment]
When playing elimination poker :
A=100
Ace of spades=1000
Joker= 500
King=10
Jump=10
Queen=10
2= 50
3= 75

**Reply by WambuaJoe** [reply]
I think hapo kwa Ace of Spades you mean 1000
Unless I'm wrong

**Reply by mwangiii** [reply]
Sisi kwetu Ace of spades is infinite,you have that you are definitely out.

**Comment by BethanyJep** [comment]
FOR BOMB NI BOMB :
😸. If it's bomb ni bomb, anything goes, you can kickback, jump a bomb with whatever shape...again, anything goes 
😸. Questions can be put however, the shape doesn't matter 
😸.  You can put a 3 or 2 at any point in the game... regardless of the shape before

**Comment by BethanyJep** [comment]
FOR BOMB NI BOMB :
😸. If it's bomb ni bomb, anything goes, you can kickback, jump a bomb with whatever shape...again, anything goes 
😸. Questions can be put however, the shape doesn't matter 
😸.  You can put a 3 or 2 at any point in the game... regardless of the shape before

**Comment by BethanyJep** [comment]
😸. The red and black joker can be used at anytime pls.

**Comment by BethanyJep** [comment]
😸. You can put a question and answer with a jump, A, kickback, bomb...etc - NO only applies kwa bomb ni bomb

**Comment by BethanyJep** [comment]
😸. The Ace of spades is used to demand a specific card eg. 7 spades♠️ , and can be blocked only by an A or a joker.

**Comment by BethanyJep** [comment]
😸. If an A of cloves ♣️ is used to block eg. a 3 of hearts♥️ we continue with the shape below the A i.e ♥'s

**Comment by BethanyJep** [comment]
![image](https://user-images.githubusercontent.com/44121227/216775090-d7754052-4c90-4318-9596-6fdb86af98fa.png)

**Comment by WambuaJoe** [comment]
Hakuna kitu ka 'bomb ni bomb kwani poker gani hii mnacheza😂
Kukulishana kadi inakuanga 
- if it's a 3 of clubs, it's either a 2 of clubs ama 3 or an A
- for the above, kama ni joker, rule huwa black on black, same goes for red
- mambo ya ku kickback ukikulishwa unafaa ufukuzwe game. Nakuwekea 3 unaniwekea K bana wacha mzaha

Also kuna ingine nimeona huwa different going by place unacheza
- Ace of Spades. Kuna watu hutumia as a double A, wengine single
- when used as a double, mtu akitumia ku cancel kulishwa si mnajua ako na extra ya kugeuza game

Alafu pia card counts during elimination
A - 100
Special A (Ace of Spades) - 1000
3 - 75
2 - 50
K, Q, J, 8, - 10
The rest ni as per the card number

Alafu mniambie, mambo ya kuweka joker kutumia Ace of Spades ku-cancel? Kwenu watu hufanya aje nishawahi sumbuliwa pale kisumu nikiwa shule 

Also, game inafaa kuenda at least round moja ndio watu waanze kuwa kadi, and also we ukizubaa unakuwa skipped. Hakuna kukumbusha mtu ni turn yake unapitwa kama game inaendelea
Nimeona kwa readme umesema stuff to do with kumaliza game, kuna ile husumbua watu ju ni kama kuelewa simple logic ni ngumu. Yes, uko cardless na uko kando yangu, but bado niko kadi because uko before me in terms of vile round inaenda. Niko legible kumaliza game if and when utapick kadi.

**Comment by VictorKabata** [comment]
If a player has a full set of winning cards (eg. three 4s) at the start of a game, each player must pick a card to slightly level the playing field.

**Reply by mwangiii** [reply]
Hii ni ya wapi sijai skia? 😂😂

**Reply by BethanyJep** [reply]
ndio naiskia leo pia 😂😂, but kuna rule ya you can't be Kadi unless everyone plays.

**Reply by Jcardif** [reply]
I think that's what @VictorKabata means, you can't be kadi 😂 unless everyone has played.

**Comment by MutayElvis** [comment]
Ukitaka kucombine a mixture of 8 na Q to use as questions, pattern ya 8 inafaa kuendana na ya Q eg 8♥️➡️8♣️➡️Q♣️➡️Q♥️➡️Q♦️➡️5♦️

**Reply by WambuaJoe** [reply]
Kabisa, hakuna kutumia 8 diamonds na Q ya flowers ju both serve the same purpose

**Comment by A-levated** [comment]
If other players have to remind you it's your turn to play, unakula fine.

**Comment by Jcardif** [comment]
Seems like we need an entire section for fines. I've taken note of the some of the common fines mentioned in the discussion forums

**Comment by Nyambura000** [comment]
mtu anisho kwenye naeza pata hii game online

**Comment by miteiyiu** [comment]
built a game with most of this rules here is the link [https://dofia.net/](url)
![image](https://github.com/user-attachments/assets/3e10436b-7b23-4995-a515-a1a8af551f44)
![image](https://github.com/user-attachments/assets/445ad312-63b2-4d29-90ae-784fb445c563)
![image](https://github.com/user-attachments/assets/afb20c4a-558f-47f1-b18a-3ce788c70f4e)
